site/components/pages/Pledge/index.tsx

```diff
@@ -44,15 +44,15 @@ export const Pledge: NextPage = () => {
         <form className={styles.inputContainer} onSubmit={handleFormSubmit}>
           <input
             className={styles.input}
-            placeholder="Search for a wallet address"
+            placeholder="Enter an address or an ENS/KNS domain"
```

Comment:
Would this be more intuitive for non-defi normies?
```suggestion
            placeholder="Enter a wallet address, .eth or .klima domain"
```